lib/src/dart_dev_runner.dart

```diff
@@ -5,20 +5,26 @@ import 'package:args/command_runner.dart';
 
 import 'dart_dev_tool.dart';
 import 'events.dart' as events;
+import 'tools/clean_tool.dart';
 import 'utils/version.dart';
 
 // import 'package:completion/completion.dart' as completion;
 
 class DartDevRunner extends CommandRunner<int> {
   DartDevRunner(Map<String, DevTool> commands)
       : super('dart_dev', 'Dart tool runner.') {
+    // For backwards-compatibility, only add the `clean` command if it doesn't
+    // conflict with any configured command.
+    commands.putIfAbsent('clean', () => CleanTool());
```

Comment:
If this is a read-only map (`const` or otherwise), this will fail; can we make a copy of the map before adding this?

Reply:
good catch!

Reply:
Or you could write it as a literal with a spread (which I guess is another way of making a copy)
`  commands = {...commands, 'clean': () => CleanTool()};`